src/natcap/invest/sdr/sdr.py

```diff
@@ -152,6 +152,21 @@
             "about": "Borselli IC0 parameter.",
             "name": "Borselli IC0 Parameter"
         },
+        "l_max": {
+            "type": "number",
+            "required": True,
+            "validation_options": {
+                "expression": "value > 0",
+            },
+            "about": (
+                "Values of L (the slope length component of the LS "
+                "slope-length gradient factor) larger than this value will "
```

Comment:
This wording is a little confusing to me. Maybe "slope length - slope gradient factor" or "slope length * slope gradient factor" to clarify that "slope length" is one thing rather than "slope * length" which is what LS looks like to me

Reply:
sure, changed in 996923d6